coriolisclient/cli/utils.py

```diff
@@ -114,3 +115,16 @@ def format_json_for_object_property(obj, prop_name):
         prop = prop.to_dict()
 
     return json.dumps(prop, indent=2)
+
+
+def validate_uuid(uuid_obj, **kwargs):
+    uuid_string = str(uuid_obj).lower()
+    uuid_version = kwargs.get('version', 4)
+    try:
+        uuid.UUID(uuid_string, version=uuid_version)
+    except ValueError:
+        # If it's a value error, then the string
+        # is not a valid hex code for a UUID.
```

Comment:
NIT: "hex code" implies all hex characters, which `-` is not.
If you really wanna get technical, the string notation is part of RFC4122 (which formally defines the byte-level layout, and also "mentions" this string notation)
That being said, it might be worth renaming the method to `validate_uuid_string` and adding a docstring saying it expects strings or string-ables which returns a UUID in RFC4122 string notation.